.github/workflows/scripts/check_wordpress_versions.php

```diff
@@ -46,15 +61,21 @@ function getLatestAndPreviousVersions($sortedVersions) {
   return [reset($latestVersionGroup), end($previousVersionGroup)];
 }
 
-function replaceLatestVersion($latestVersion) {
+/**
+ * We specify the latest WordPress version only in the docker-compose file for the tests.
+ */
+function replaceLatestVersion($latestVersion): void {
```

Comment:
I see that you don't use typehints for method parameters. Here but also for other functions. Is that on purpose? I think they can be helpful to catch error when something (e.g. API response) changes.